src/components/Modals/OnboardingReview.vue

```diff
@@ -0,0 +1,514 @@
+                >
+              </div>
+              <label class="onboarding-label addSpaceTop">Score</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'IBScore' + index">
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'IB'"
+                  class="onboarding-label--review"
+                  >{{ options.score.placeholder }}</label
+                >
+              </div>
+            </div>
+            <div class="alignCenter">
+              <label class="onboarding-label">Credit</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'APCredit' + index">
+                <!-- TODO replace credit with actual value -->
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'AP'"
+                  class="onboarding-label--review"
+                  >{{ getExamCredit(options) }}</label
+                >
+              </div>
+              <label class="onboarding-label addSpaceTop">Credit</label>
+              <div v-for="(options, index) in displayOptions.exam" :key="'IBCredit' + index">
+                <label
+                  v-if="typeof options.type != undefined && options.type.placeholder == 'IB'"
+                  class="onboarding-label--review"
+                  >{{ getExamCredit(options) }}</label
+                >
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="onboarding-subHeader2-fillRow">
+          <span class="onboarding-subHeader2-review"> Transferred Course Credits</span>
+        </div>
+        <div class="onboarding-selectWrapper">
+          <div class="onboarding-selectWrapper-reviewExam">
+            <div>
+              <div v-for="(options, index) in displayOptions.class" :key="index">
+                <label v-if="options.class !== 'Select one'" class="onboarding-label--review">
+                  {{ options.class }}
+                </label>
+              </div>
+            </div>
+            <div class="alignEnd">
+              <div v-for="(options, index) in displayOptions.class" :key="index">
+                <label v-if="options.class !== 'Select one'" class="onboarding-label--review">
+                  {{ options.credits }} Credits
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="onboarding-bottomWrapper">
+          <label class="onboarding-subHeader2-review">Total Transfer Credits:</label>
+          <div class="onboarding-label--bottom">
+            <label class="onboarding-label--bottom---bold">{{ totalCredits }}</label>
+            <label>Credits</label>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+// TODO: a lot of the functions are repeated from basic and transfer pages, is there a way to avoid that?
```

Comment:
Yes! We can make a separate ts file that is imported into all 3 of these components. This is actually a really great idea... 

Can you delete this comment, then leave a note in your PR that this should be done in the future, and make a Monday item for it?